lib/api/apiUtils/bucket/models/ReplicationConfiguration.js

```diff
@@ -101,14 +121,13 @@ class ReplicationConfiguration {
             prefix: rule.Prefix[0],
             enabled: rule.Status[0] === 'Enabled',
         };
+        // ID is an optional property, but create one if not provided or is ''.
+        obj.id = rule.ID && rule.ID[0] !== '' ? rule.ID[0] :
```

Comment:
If `rule.ID` is defined, it will be an array with one element that is a string, since it comes from the  `xml2js.parseString` method.